12_6_Accessing_Bitcoind_with_Python.md

```diff
@@ -0,0 +1,86 @@
+# 12.6: Accessing Bitcoind with Python
+
+> **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+Though command-line `curl` is the easiest way to access the `bitcoind` directly, there are many other options for doing so and most of them support more fully featured programming languages. One package for doing so in Python is [Bitcoin authproxy](https://github.com/bitcoin/bitcoin/blob/master/test/functional/test_framework/authproxy.py). It was originally [written by Jeff Garzik](https://github.com/jgarzik/python-bitcoinrpc) but is now maintained by Bitcoin Core as the primary interface for the QA testing suite. It uses an http client module to access the `bitcoind` RPC server.
+
+## Set Up Bitcoin AuthProxy
+
+The latest Bitcoin Core version of `authproxy` requires Python 3, but you can still use jgarzik's original script to run in Python 2. Download the script to your working directory, start Python, and import the module. This example uses Python 3 running on Linux:
```

Comment:
If possible, we're trying to include the complete instructions for how to use these Bitcoin options, which means here we'd like to show the complete methodology for getting the Python installed and setup. Ideally, use the same Debian setup that we use for most of our examples.